SocialFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Extended CSS rules
 !
+washingtonpost.com#?#div[aria-controls="share-tooltip"]:upward(1)
```

Comment:
Can by hidden by `washingtonpost.com##div[data-qa="utility-bar"] > div[data-sc-c="tooltip"]` Added it.